lib/resque/worker.rb

```diff
@@ -503,17 +503,15 @@ def heartbeat!(time = Time.now)
     end
 
     def start_heartbeat
-      heartbeat!
```

Comment:
As far as I understand it, we aren't pruning if you don't have a heartbeat, we only prune if there is a heartbeat but it expired (i.e. the key exists but the timestamp is too old).